### PR TITLE
Implement the authorization between client and server

### PR DESCRIPTION
**Pull request by kamontia**
image

### CODE REVIEW

File: app/javascript/components/TodoComponent.tsx [hunk 1]
```diff
@@ -9,19 +12,35 @@ interface IProps {
...
     this.state = {
       text: '',
-      todos: []
+      todos: [],
+      status: false
     }
   }
 
+  componentWillMount = () => {
+    const URL = 'http://localhost:3000/todos/auth_check'
```
Comment (chaspy) [inline app/javascript/components/TodoComponent.tsx]: Does it work in production?

File: app/controllers/todos_controller.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class TodosController < ApplicationController
-  before_action :set_todo, only: [:show, :edit, :update, :destroy]
+  before_action :set_todo, only: [:show, :edit, :update, :destroy], except: [:auth_check]
```
Comment (chaspy) [inline app/controllers/todos_controller.rb:2]: No todos_controller#auth_check, is it?

File: app/javascript/components/TodoComponent.tsx (hunk 1 above)
Comment (kamontia) [reply app/javascript/components/TodoComponent.tsx]: Fixed

File: app/controllers/todos_controller.rb (hunk 2 above)
Comment (kamontia) [reply app/controllers/todos_controller.rb]: Yes, remove the filter

File: app/javascript/components/TodoComponent.tsx (hunk 1 above)
Comment (chaspy) [reply app/javascript/components/TodoComponent.tsx]: 6912351

File: app/controllers/todos_controller.rb (hunk 2 above)
Comment (chaspy) [reply app/controllers/todos_controller.rb]: 504c9ae

**Pull request merged**